utils/api/api.go

```diff
@@ -73,3 +74,30 @@ func ParseBody[T any](body any) (*T, error) {
 	}
 	return entity, nil
 }
+
+func ResolveRouteParams(pattern, routes string) (map[string][]string, bool) {
+	pTokens := strings.Split(pattern, ".")
+	sTokens := strings.Split(routes, ".")
+
+	if len(pTokens) != len(sTokens) {
+		return nil, false
+	}
+	result := make(map[string][]string)
+
+	for i := 0; i < len(pTokens); i++ {
+		switch pTokens[i] {
+		case "*":
+			if i > 0 {
+				key := pTokens[i-1]
+				result[key] = []string{sTokens[i]}
+			} else {
+				result["*"] = []string{sTokens[i]}
+			}
+		default:
+			if pTokens[i] != sTokens[i] {
+				return nil, false
+			}
+		}
+	}
+	return result, true
+}
```

Comment:
Let's write a test case as well.

I think this doesn't work for such a case:

- `get.api.point.name.*.*.*` // here we want three route params

In such case, maybe we may need to define the pattern as:

`get.api.point.name.:network_name.:device_name.:point_name`